sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -724,11 +729,14 @@ public Key restoreKey(byte[] backup) {
      * call {@link KeyClient#getKey(KeyBase baseKey)} . This will return the {@link Key key} with key material included of its latest version.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listKeys#Context}
      *
+     * <p><strong>Samples to iterate KeyBase by each page</strong></p>
```

Comment:
The sample description needs more description as to what its doing like the one above it, +  its doing it per page.